root-packages/cvmfs/0003-crosscompile-sha3.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/externals/sha3/src/makeHook.sh b/externals/sha3/src/makeHook.sh
+index 7743eb57e..851f83b04 100755
+--- a/externals/sha3/src/makeHook.sh
++++ b/externals/sha3/src/makeHook.sh
+@@ -1,5 +1,7 @@
+ #!/bin/sh
+ 
++echo "64compact" > arch
```

Comment:
The default value was giving errors atleast on arm64, with this it worked on both arm64 and x86_64.

Not sure if this could be generalized